### PR TITLE
[quoteservice] manual metrics, logs export

### PR DESCRIPTION
**Pull request by brettmc**
# Changes

- add a manual metric to the php quote service
- randomize per-item cost to add some variability to quotes
- bump dependencies to latest
- fix a monolog deprecation
- ensure logs are exported per the configured delay

## Merge Requirements

For new features contributions please make sure you have completed the following
essential items:

* [x] `CHANGELOG.md` updated to document new feature additions
* [ ] Appropriate documentation updates in the [docs][]
* [ ] Appropriate Helm chart updates in the [helm-charts][]

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

[docs]: https://opentelemetry.io/docs/demo/
[helm-charts]: https://github.com/open-telemetry/opentelemetry-helm-charts

### CODE REVIEW

**Comment by brettmc** [comment]
Related to https://github.com/open-telemetry/opentelemetry.io/issues/2669

File: src/quoteservice/app/routes.php [hunk 1]
```diff
@@ -26,12 +26,20 @@ function calculateQuote($jsonObject): float
...
         $childSpan->setAttribute('app.quote.cost.total', $quote);
 
         $childSpan->addEvent('Quote calculated, returning its value');
+
+        //manual metrics
+        static $counter;
+        $counter ??= Globals::meterProvider()
+            ->getMeter('quotes')
+            ->createCounter('quotes', 'quotes', 'number of quotes calculated');
+        $counter->add(1, ['number_of_items' => $numberOfItems, 'total_cost' => $quote]);
```
Comment (puckpuck) [inline src/quoteservice/app/routes.php:42]: The `total_cost` attribute here will create a cardinality explosion on the metric. One could also argue the `number_of_items` tag could do the same, but I feel like that number won't go too wild, and it makes sense to leave it so people can see how you add tags to metrics.

**Review by puckpuck — CHANGES_REQUESTED** [review]

File: src/quoteservice/app/routes.php (hunk 1 above)
Comment (brettmc) [reply src/quoteservice/app/routes.php]: removed total cost attribute

**Review by puckpuck — APPROVED** [review]

**Review by puckpuck — APPROVED** [review]

**Comment by brettmc** [comment]
> Since the returned cost is now randomized

Thanks, I didn't consider if there might be downstream effects of randomizing the cost.

**Review by julianocosta89 — APPROVED** [review]

**Pull request merged**